tools/expotools/src/versioning/ios/index.ts

```diff
@@ -533,36 +536,35 @@ async function generatePodfileDepsAsync(
     versionedPodNames.React,
     globals.React.concat(globals.yoga)
   );
-  const indent = '  '.repeat(6);
-  let config = `
-      # Generated postinstall: ${versionedPodNames.React}
-      if pod_name == '${versionedPodNames.React}' || pod_name == '${versionedPodNames.ExpoKit}'
-      target_installation_result.native_target.build_configurations.each do |config|
-          config.build_settings['OTHER_CFLAGS'] = %w[
-            ${configValues.join(`\n${indent}`)}
-          ]
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)']
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_DEV=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_ENABLE_INSPECTOR=0'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}ENABLE_PACKAGER_CONNECTION=0'
-          # Enable Google Maps support
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}HAVE_GOOGLE_MAPS=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}HAVE_GOOGLE_MAPS_UTILS=1'
-        end
-      end
-      # End generated postinstall`;
+  const indent = '  '.repeat(3);
+  const config = `# @generated by expotools
+      
+if pod_name == '${versionedPodNames.React}' || pod_name == '${versionedPodNames.ExpoKit}'
+  target_installation_result.native_target.build_configurations.each do |config|
+    config.build_settings['OTHER_CFLAGS'] = %w[
+      ${configValues.join(`\n${indent}`)}
+    ]
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)']
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_DEV=1'
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}RCT_ENABLE_INSPECTOR=0'
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}ENABLE_PACKAGER_CONNECTION=0'
+    # Enable Google Maps support
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}HAVE_GOOGLE_MAPS=1'
+    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << '${versionName}HAVE_GOOGLE_MAPS_UTILS=1'
+  end
+end
+`;
   await fs.writeFile(
-    path.join(
-      templatesPath,
-      'versioned-react-native',
-      'postinstalls',
-      `${versionedPodNames.React}.rb`
-    ),
-    `${config}\n`
+    path.join(versionedReactPodPath, 'postinstalls.rb'),
+    config,
   );
   return;
 }
 
+/**
+ * TODO(@tsapeta): This function can be removed once we drop SDK35,
+ * this is the last SDK that has versioned templates in template-files folder.
```

Comment:
It looks like you moved versioned templates to `ios/versioned-react-native`, are you sure this function has to remain here?